charts/incubator/teslamate/values.yaml

```diff
@@ -0,0 +1,114 @@
+  repository: teslamate/grafana
+  tag: latest
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+
+service:
+  main:
+    ports:
+      main:
+        protocol: http
+        targetPort: 4000
+        port: 4000
+  grafana:
+    enabled: true
+    type: ClusterIP
+    ports:
+      grafana:
+        enabled: true
+        protocol: http
+        targetPort: 3000
+        port: 3000
+
+workload:
+  main:
+    replicas: 1
+    strategy: RollingUpdate
+    podSpec:
+      containers:
+        main:
+          env:
+            ENCRYPTION_KEY:
+              secretKeyRef:
+                name: teslamate-secrets
+                key: TESLAMATE_ENCRYPTION_KEY
+            DATABASE_USER: "{{ .Values.cnpg.main.user }}"
+            DATABASE_PASS:
+              secretKeyRef:
+                name: cnpg-main-user
+                key: password
+            DATABASE_NAME: "{{ .Values.cnpg.main.database }}"
+            DATABASE_HOST:
+              secretKeyRef:
+                name: cnpg-main-urls
+                key: host
+            MQTT_HOST: teslamate-mosquitto
+            GF_SECURITY_ADMIN_USER: "admin"
+            GF_SECURITY_ADMIN_PASSWORD: "testpassword"
+          probes:
+            liveness:
+              path: "/"
+            readiness:
+              path: "/"
+            startup:
+              path: "/"
+        grafana:
+          enabled: true
+          imageSelector: grafanaImage
+          env:
+            DATABASE_USER: "{{ .Values.cnpg.main.user }}"
+            DATABASE_PASS:
+              secretKeyRef:
+                name: cnpg-main-user
+                key: password
+            DATABASE_NAME: "{{ .Values.cnpg.main.database }}"
+            DATABASE_HOST:
+              secretKeyRef:
+                name: cnpg-main-urls
+                key: host
+            GRAFANA_PASSWD: "{{ .Values.workload.main.podSpec.containers.main.env.GF_SECURITY_ADMIN_PASSWORD }}"
+            GF_SECURITY_ADMIN_USER: "{{ .Values.workload.main.podSpec.containers.main.env.GF_SECURITY_ADMIN_USER }}"
+            GF_SECURITY_ADMIN_PASSWORD: "{{ .Values.workload.main.podSpec.containers.main.env.GF_SECURITY_ADMIN_PASSWORD }}"
+          probes:
+            liveness:
+              port: "{{ .Values.service.grafana.ports.grafana.port }}"
+              path: "/api/health"
+            readiness:
+              port: "{{ .Values.service.grafana.ports.grafana.port }}"
+              path: "/api/health"
+            startup:
+              port: "{{ .Values.service.grafana.ports.grafana.port }}"
+              path: "/api/health"
+
+mosquitto:
+  enabled: true
+  ingress:
+    websockets:
+      enabled: false
+
+persistence:
+  grafana-data:
+    enabled: true
+    mountPath: "/var/lib/grafana"
+
+portal:
+  open:
+    enabled: true
+
+cnpg:
+  main:
+    enabled: true
+    user: teslamate
+    database: teslamate
+    cluster:
+      initdb:
+        postInitApplicationSQL:
+          - ALTER USER teslamate WITH SUPERUSER
```

Comment:
```suggestion
cnpg:
  main:
    enabled: true
    user: teslamate
    database: teslamate
```

Reply:
Teslamate runs migrations on launch and upgrade which require superuser access. I tried to manually install the extensions but then it needed it for other things so i thought to ensure initial and future migrations work without issues this was the best way.

Reply:
@xstar97 I think this is a relatively neat way to deal with the issue.
It would never be permissable in the premium train, but for stable/incubator I think it's relatively okey for applications that are inherently badly designed